bot/define_command.go

```diff
@@ -0,0 +1,57 @@
+package bot
+
+import (
+	"fmt"
+	"io"
+	"net/url"
+	"strings"
+
+	"github.com/urfave/cli"
+	"github.com/zpatrick/rclient"
+)
+
+// DatamuseAPIEndpoint for the datamuse api
+const DatamuseAPIEndpoint = "https://api.datamuse.com"
+
+// DatamuseResponse is the response type for a Tenor API search
+type DatamuseResponse []struct {
```

Comment:
This looks pretty odd - I'm pretty confident it's not best practice to declare types of `[]struct`. 
It's usually up to the caller to declare a slice or not. 

I believe this will still work with:
```go
type DatamuseResponse struct {
    Definitions []string `json:"defs"`
    Word        string   `json:"word"`
}
```

and below
```go
var response []DatamuseResponse
if err := client.Get(...
```

Reply:
Okay sweet. Yeah the response isn't typically for muse. I had to add this to get a response properly. I like that option better though. 

```
var response []DatamuseResponse
if err := client.Get(...
```
^ I'll go with that.